src/frontend/components/MobileFilter/MobileFilter.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./MobileFilter.css";
+import { useAccordion } from "../../contexts/AccordionContext/AccordionContext";
+const MobileFilter = () => {
+  const { setIsMobileFilter } = useAccordion();
+  return (
+    <div className="mobile-filter">
+      <button
+        className="mobile-filter-btn ct-gray-bd ct-btn"
+        onClick={() => setIsMobileFilter((isTrue) => (isTrue ? false : true))}
```

Comment:
If you want to toggle the state here you could just negate it instead of using ternary.

Reply:
Ohh yes Thank you Brother